src/deadline_worker_agent/log_messages.py

```diff
@@ -386,13 +391,14 @@ class SessionLogEventSubtype(str, Enum):
     COMPLETE = "Complete"
     INFO = "Info"  # Generic information about the session
     LOGS = "Logs"  # Info on where the logs are going
+    OPENJD = "OpenJD"  # Logs from the openjd.sessions module applicable to the worker log
```

Comment:
I'd suggest a different subtype. Something more generic?

That these messages originate from the `openjd-sessions` library is an implementation detail that end-users will, I think, not care about nor understand the nuances of why the detail is important.

e.g. `Runtime` -- Information about the processes that are running in the Session.

Reply:
or `ProcessControl` maybe?

Reply:
`ProcessControl` might not always be correct (specifically for messages which just contain exception or host info)

`Runtime` could work though

Reply:
Added in the latest commit